book/src/configuration.md

```diff
@@ -1,5 +1,12 @@
 # Configuration
 
+## LSP
+
+To disable language server progress report from being displayed in the status bar add this option to your `config.toml`:
+```toml
+lsp_progress = false
```

Comment:
Sorry, can you change it to `lsp-progress`? We match the kebab-case convention used in Cargo

Reply:
Done.